src/factory.ts

```diff
@@ -464,10 +464,11 @@ export class Factory {
     return fingering;
   }
 
-  StringNumber(params: { number: string; position: string }): StringNumber {
+  StringNumber(params: { number: string; position: string; drawCircle?: boolean }): StringNumber {
```

Comment:
It's not clear from this if the circle is drawn or not by default. Can you make it `drawCircle: boolean = true`? (You can remove the `if undefined` below too, in that case.)

Reply:
This has to be done outside params, done.